src/Powercord/migrations/20220822-move-themes-plugins.js

```diff
@@ -0,0 +1,34 @@
+const fs = require('fs');
+const { join } = require('path');
+
+module.exports = () => {
+  const oldPlugs = join(__dirname, '..', 'plugins');
+  const newPlugs = join(__dirname, '..', '..', '..', 'plugins');
+  const oldThemes = join(__dirname, '..', 'themes');
+  const newThemes = join(__dirname, '..', '..', '..', 'themes');
+
+  const rmdirOpt = {recursive: true}
+
+  // This function recursively moves all subdirectories and files from src to dest
+  let moveRecursiveSync = function(src, dest) {
+    const exists = fs.existsSync(src);
+    const stats = exists && fs.statSync(src);
+    const isDirectory = exists && stats.isDirectory();
+    if (isDirectory) {
+      try {
+        fs.mkdirSync(dest);
+      } catch (e) {}
+      fs.readdirSync(src).forEach(function(childItemName) {
+        moveRecursiveSync(join(src, childItemName),
+                          join(dest, childItemName));
+      });
+    } else {
+      fs.renameSync(src, dest);
+    }
```

Comment:
Think this can be much simpler
```suggestion
    fs.readdirSync(src).forEach(function(childItemName) {
      fs.renameSync(src, dest);
    });
```